src/components/Shared/AddressTab.js

```diff
@@ -163,20 +159,20 @@ const AddressTab = ({
                 { key: 'flName', value: 'Foreign Language Name' }
               ]}
               values={addressValidation.values}
-              required
+              required={!requiredOptional}
               onChange={(event, newValue) => {
-                setCityStore([])
-                addressValidation.setFieldValue('stateId', null)
-                addressValidation.setFieldValue('cityId', null)
-                addressValidation.setFieldValue('city', null)
-                addressValidation.setFieldValue('cityDistrictId', null)
-                addressValidation.setFieldValue('cityDistrict', null)
-
+                addressValidation.setFieldValue('stateId', 0)
+                addressValidation.setFieldValue('cityId', '')
+                addressValidation.setFieldValue('city', '')
```

Comment:
Please check if cityId and cityDistrictId are enough. no need for city and cityDistrict fields